apps/home/src/components/HomePageGrid/styles/HomePageGrid.module.scss

```
@@ -0,0 +1,52 @@
+  0% {
+    background-position: 200% 0;
+  }
+
+  100% {
+    background-position: -200% 0;
+  }
+}
+
+.emptyState {
+  @include type-style("label-01");
+  text-align: center;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)